progect-management-app/src/app/components/main-boards/main-boards.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="boards_container">
+  <div class="board_head">
+  <h2 class="board_main-title">Boards</h2>
+  <button (click)="addBoard()" style="width: 200px; border-radius: 10px" mat-mini-fab matTooltip="Add board" color="primary">
+    <mat-icon>add</mat-icon>
+  </button>
+  </div>
+  <div class="board_container">
+  <div class="board" *ngFor="let board of boards">
+    <div>
+      <p *ngIf="selectedBoard !== board" >{{ board.title }}</p>
+      <div class="updateField" *ngIf="selectedBoard === board">
```

Comment:
It better to have all logic in the component file. Let's imagine that you should add some piece of logic here. So you have to refactor code and only after that you would be able to add new logic.